package.json

```
@@ -0,0 +1,156 @@
+{
+  "name": "react-survey",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@babel/core": "7.12.3",
```

How about adding [Nimble ESLint config](https://www.npmjs.com/package/@nimblehq/eslint-config-nimble) package?

```suggestion
    "@babel/core": "7.12.3",
    "@nimblehq/eslint-config-nimble": "2.1.1",
```

---

I will work on it tomorrow~ I need to see how to include it in the config~ Thanks! :)

---

@carryall Ok, I've implemented it in the last commits. 
Quite many changes as I also implemented 
- stylelint with a git pre-commit hook (prevent to commit scss code with errors)
- integrate eslint in the compilation process (instead of as an external ~ manual command) — so that errors/warnings are imediates
- updated the README.md file accordingly

And of course tested warnings/errors and compliance with Nimble Compass conventions :) 
PR description also updated~ 

It should be good now :)